tests/components/ClaimStatus.test.tsx

```diff
@@ -56,7 +57,9 @@ describe('Scenario 1', () => {
 
 describe('Scenario 2', () => {
   beforeAll(() => {
-    MockDate.set('2021-05-05')
+    Settings.defaultZone = 'America/Los_Angeles'
+    const expectedNow = DateTime.local(2021, 5, 5)
+    Settings.now = () => expectedNow.toMillis()
```

Comment:
Harness the luxon Settings to override `.now` - neat

[Based on this guide](https://dev.to/thealexkates/how-to-mock-luxon-s-datetime-now-when-using-tdd-3j6o)